bioconda_utils/cli.py

```diff
@@ -98,6 +103,24 @@ def build(recipe_folder,
     else:
         docker_builder = None
 
+    # handle git range
+    if git_range:
+        p = sp.run(['git', 'diff',
+                          '--relative=recipes',
+                          '--name-only'] + git_range +
+                         [os.path.join(recipe_folder, '*', 'meta.yaml'),
```

Comment:
@johanneskoester if we leave this out here and consider all files in the `run` function and filter later by `modified = [mod for mod in modified if mod.endswith('mata.yaml')]` this should be more stable I assume.